### PR TITLE
Add r-rJava to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
Work completed for:

- [x] Linux
- [x] OSX

Skipped for:
- [x] Windows

### CODE REVIEW

File: recipes/r-rjava/build.sh [hunk 1]
```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
 
 # This is a little bit of cheating, the correct fix is in r-base
-sed -i 's/~autodetect~/-Wl,-rpath,$JAVA_LD_LIBRARY_PATH -L$JAVA_LD_LIBRARY_PATH -ljvm/' $PREFIX/lib/R/etc/javaconf
+if [ $(uname) = 'Darwin' ]; then
+	sed -i 's|JAVA_CPPFLAGS=~autodetect~|JAVA_CPPFLAGS=-I$JAVA_HOME/include -I$JAVA_HOME/include/darwin|' \
```
Comment (bgruening) [inline recipes/r-rjava/build.sh]: The BSD version should work if you use `sed -i.bak ''`

**Comment by sodre** [comment]
I need to apply the hack for windows through the bold.bat file. I will do that later today.

**Comment by sodre** [comment]
@johanneskoester 
   I will let you decide if you want to merge this one, or if you rather merge my changes back to your original PR. If you merge it back, can I be kept as a maintainer ?

**Comment by sodre** [comment]
Yes, the hack should be obsolete after the new r-base package is available in anaconda. I will undo the hack for rJava in my PR if you rather merge r-openssl and r-rJava separately.

Patrick

**Comment by johanneskoester** [comment]
Ok!

**Comment by sodre** [comment]
@johanneskoester,
  alright, everything should be fine from the here since the AppVeyor job will be skipped. Feel free to merge once we have all three green lights.

On a separate note, are you able to help me get "reviewer" status for conda-forge/staged-recipes? It would help me out a lot.

**Comment by johanneskoester** [comment]
Thanks. I can only forward your request to @jakirkham. @jakirkham, @sodre was a great help with solving r and Java issues. I am sure his expertise could be very helpful for conda-forge.

I don't know your policies though.

**Pull request merged**